cypress/e2e/yves/ssp-service/ssp-service.cy.ts

```diff
@@ -0,0 +1,178 @@
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+                // Sort by Created At
+                sspServiceListPage.clickSortColumn('Created At');
+
+                // Verify that sorting was triggered
+                sspServiceListPage.getOrderByInput().should('have.value', 'created_at');
+                sspServiceListPage.getOrderDirectionInput().should('have.value', 'ASC');
+
+                // Click again to toggle sort direction
+                sspServiceListPage.clickSortColumn('Created At');
+
+                // Verify sort direction was toggled
+                sspServiceListPage.getOrderByInput().should('have.value', 'created_at');
+                sspServiceListPage.getOrderDirectionInput().should('have.value', 'DESC');
+            });
+
+            it('should search services by SKU', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                });
+                prepareServices();
+                // Get product SKU from fixtures to search for
+                const productSku = dynamicFixtures.product1.sku;
+
+                // Select SKU search type and enter the product SKU
+                sspServiceListPage.searchFor('SKU', productSku);
+
+                // Wait for the search results to load
+                cy.wait(500);
+
+                // Verify search filter is applied in URL
+                cy.url().should('include', productSku);
+
+                // Verify exactly one row is found (the exact match)
+                sspServiceListPage.getTableRows().should('have.length', 1);
+            });
+
+            it('customer without permissions should se only his own services', (): void => {
+                customerLoginScenario.execute({
+                    email: dynamicFixtures.customer2.email,
+                    password: staticFixtures.defaultPassword,
+                    withoutSession: true,
+                })
+                prepareServices();
+
+                // Check if Business Unit dropdown exists
+                sspServiceListPage.getBusinessUnitSelect().should('exist');
+
+                // Check that 'My Services' option is available in the dropdown
+                sspServiceListPage.getBusinessUnitSelect().find('option').should('contain', 'My Services');
+
+                // A regular customer should not see business unit and company claims
+                sspServiceListPage.getBusinessUnitSelect().find('option[value*="businessUnit"]').should('not.exist');
+                sspServiceListPage.getBusinessUnitSelect().find('option[value*="company"]').should('not.exist');
+
+                sspServiceListPage.getTableRows().should('not.exist');
+            });
+        });
+    },
+);
```

Comment:
We had an issue in the past with Claims, when comnay user was able to see claims from other companies. 
Please add a test that company user can see purchased Services only from his company/business unit: 
- prepare: customer A and B from company A and company B exists in the system. Place 2 orders by these customers
- execute: filter purchased services by Company or Business unit by customer A
- assert: customer A can not see Services from company B